xcp_d/workflow/anatomical.py

```diff
@@ -1,784 +1,1371 @@
+        )[0]
+        L_pial_surf = fnmatch.filter(
+            all_files, "*sub-*" + subject_id + "*hemi-L_pial.surf.gii"
+        )[0]
+        R_pial_surf = fnmatch.filter(
+            all_files, "*sub-*" + subject_id + "*hemi-R_pial.surf.gii"
+        )[0]
+        L_wm_surf = fnmatch.filter(
+            all_files, "*sub-*" + subject_id + "*hemi-L_smoothwm.surf.gii"
+        )[0]
+        R_wm_surf = fnmatch.filter(
+            all_files, "*sub-*" + subject_id + "*hemi-R_smoothwm.surf.gii"
+        )[0]
+
+        ribbon = fnmatch.filter(
+            all_files, "*sub-*" + subject_id + "*desc-ribbon.nii.gz"
+        )[0]
+
+        ses_id = _getsesid(ribbon)
+        anatdir = output_dir + "/xcp_d/sub-" + subject_id + "/ses-" + ses_id + "/anat"
+        if not os.path.exists(anatdir):
+            os.makedirs(anatdir)
+
+        surf = [
+            L_inflated_surf,
+            R_inflated_surf,
+            L_midthick_surf,
+            R_midthick_surf,
+            L_pial_surf,
+            R_pial_surf,
+            L_wm_surf,
+            R_wm_surf,
+        ]
+
+        for ss in surf:
+            shutil.copy(ss, anatdir)
+
+        ribbon2statmap_wf = pe.Node(
+            RibbontoStatmap(ribbon=ribbon),
+            name="ribbon2statmap",
+            mem_gb=mem_gb,
+            n_procs=omp_nthreads,
+        )
+
+        brainspritex_wf = pe.Node(
+            BrainPlotx(), name="brainsprite", mem_gb=mem_gb, n_procs=omp_nthreads
+        )
+        from ..utils import ContrastEnhancement
+
+        enhancet1w_wf = pe.Node(
+            ContrastEnhancement(),
+            name="enhancet1w",
+            mem_gb=mem_gb,
+            n_procs=omp_nthreads,
+        )
+        ds_brainspriteplot_wf = pe.Node(
+            DerivativesDataSink(
+                base_directory=output_dir,
+                check_hdr=False,
+                dismiss_entities=["desc"],
+                desc="brainplot",
+                datatype="figures",
+            ),
+            name="brainspriteplot",
+            run_without_submitting=True,
+        )
+
+        workflow.connect(
+            [
+                (ribbon2statmap_wf, brainspritex_wf, [("out_file", "in_file")]),
+                (inputnode, enhancet1w_wf, [("t1w", "in_file")]),
+                (enhancet1w_wf, brainspritex_wf, [("out_file", "template")]),
+                (brainspritex_wf, ds_brainspriteplot_wf, [("out_html", "in_file")]),
+                (inputnode, ds_brainspriteplot_wf, [("t1w", "source_file")]),
+            ]
+        )
+
+    else:
+        all_files = list(layout.get_files())
+
+        t1w_transform_wf = pe.Node(
+            ApplyTransformsx(
+                num_threads=2,
+                reference_image=mnitemplate,
+                # transforms=[str(t1w_to_mni), str(MNI92FSL)], #TM: need to replace MNI92FSL xfm with the correct xfm from the MNI output space of fMRIPrep/NiBabies (MNI2009, MNIInfant, or for cifti output MNI152NLin6Asym) to MNI152NLin6Asym.

```

Comment:
@madisoth did you add this comment or was it in there before? Should we open a PR for this?